<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlagsWidget</class>
 <widget class="QDockWidget" name="FlagsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Flags</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTreeView" name="flagsTreeView">
      <property name="styleSheet">
       <string notr="true">QTreeView::item
{
    padding-top: 1px;
    padding-bottom: 1px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="indentation">
       <number>8</number>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <property name="animated">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_17">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLineEdit" name="filterLineEdit">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Quick Filter</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="flagspaceLabel">
        <property name="text">
         <string>Flagspace:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="flagspaceCombo"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="actionRename">
   <property name="text">
    <string>Rename</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
